pkgs/os-specific/linux/trelay/Makefile

```diff
@@ -0,0 +1,14 @@
+KERNELRELEASE ?= $(shell uname -r)
```

Comment:
We can't use https://github.com/openwrt/openwrt/blob/main/package/kernel/trelay/Makefile ?